R-packages/evalcast/vignettes/intro_vignette.Rmd

```diff
@@ -0,0 +1,132 @@
+                                     geo_type = "state")
+```
+
+`baseline_forecaster` created a list of `r length(predictions_cards)` predictions cards, (one for each `(forecast date, ahead)` pair).  Let's look at a predictions card:
+
+```{r}
+predictions_cards[[1]]
+```
+
+Forecasts are probabilistic, meaning that rather than predicting a single value, a distribution of values is generated. For example, the baseline forecaster would have predicted the following distribution in California for `r attributes(predictions_cards_cmu[[1]])$ahead` week(s) after `r attributes(predictions_cards_cmu[[1]])$forecast_date`
+```{r}
+pc <- predictions_cards_cmu[[1]]
+ca_fips = covidcast::name_to_fips("California")
+ca_code = substr(ca_fips, 0, 2) # States are identified by a 2 character code
+pc[pc$location == ca_code, ]$forecast_distribution[[1]]
+```
+
+### Evaluate the performance of a forecaster
+
+Now that we've made these predictions, we'd like to know how well they perform.  By default, `evalcast` uses the following three performance measures, but these can be easily substituted:
+
+```{r}
+err_measures <- list(wis = weighted_interval_score,
+                     ae = absolute_error,
+                     coverage_80 = interval_coverage(alpha = 0.2))
+```
+
+We may now create scorecards for each forecaster's predictions or for a filtered subset. 
+
+```{r}
+scorecards_yyg <- evaluate_predictions(filter_predictions(predictions_cards_yyg, ahead = 3), 
+                                   err_measures, 
+                                   backfill_buffer = 10)
+pcards_cmu_death3 <- filter_predictions(predictions_cards_cmu, 
+                                        ahead = 3,
+                                        response_data_source = "jhu-csse",
+                                        response_signal = "deaths_incidence_num")
+scorecards_cmu <- evaluate_predictions(pcards_cmu_death3, 
+                                   err_measures, 
+                                   backfill_buffer = 10)
+scorecards <- evaluate_predictions(filter_predictions(predictions_cards, ahead = 3), 
+                                   err_measures, 
+                                   backfill_buffer = 10)
+```
+
+Let's look at the `baseline_forecaster`'s scorecard for 3-epiweek ahead forecasts.
+
+```{r}
+scorecards[[1]]
+```
+
+**Technical note:** What does `backfill_buffer = 10` do?  When we evaluate a forecaster in backtesting we are assuming that we know what actually occurred.  However, in light of backfill, we may not trust the data for some period of time.  The argument `backfill_buffer` allows us to specify how many days until we believe that the data has "settled down" and is unlikely to be updated further.  The choice of this argument will depend on the particular signal you are forecasting.
+
+### Plots of performance measures
+
+`evalcast` provides a number of tools to visually assess a forecaster (including comparisons with other forecasters):
+
+```{r}
```

Comment:
These plots all look truncated or squished.  I think you could use the `fig.width` and `fig.height` code chunk options to make this look better.